modules/RTC/JitsiLocalTrack.js

```diff
@@ -101,6 +107,19 @@ function JitsiLocalTrack(stream, track, mediaType, videoType, resolution,
 
     RTCUtils.addListener(RTCEvents.DEVICE_LIST_CHANGED,
         this._onDeviceListChanged);
+
+    if(this.isVideoTrack() && this.videoType === VideoType.CAMERA) {
+        this._setHandler("track_mute", function () {
+            if(!this._checkForCameraIssues())
```

Comment:
Adding return here on reversed condition looks like unnecessary complication and is less readable. I'd rather see it as:
```
if (this._checkForCameraIssues() {
   this.eventEmitter.emit(JitsiTrackEvents.NO_DATA_FROM_SOURCE);
}
```